blackbox/docs/sql/aggregation.txt

```diff
@@ -373,6 +373,36 @@ strings.  The advantage is that the ``arbitrary`` function does very little
 to no computation as for example ``max`` aggregation function would
 do.
 
+
+percentile_cont
+===============
+
+The ``percentile_cont`` aggregation function, when provided a percentage and a
+sorted set, will return a single value above/below which the given 
+percentage of values in the sorted set reside.
```

Comment:
suggestion: try to make the sentence less complicated

make 2 sentences for single value and array value input